src/components/content/content.test.js

```diff
@@ -0,0 +1,80 @@
+import * as React from "react";
+import Content from "./content.component";
+import CypressMountWithProviders from "../../../cypress/support/component-helper/cypress-mount";
+
+import {
+  contentTitle,
+  contentBody,
+  contentPreview,
+} from "../../../cypress/locators/content/index";
+
+context("Tests for Content component", () => {
+  describe("should check Content component properties", () => {
+    it.each(["mp150ú¿¡üßä", "!@#$%^*()_+-=~[];:.,?{}&\"'<>"])(
```

Comment:
we could use this `each` values as a const as usual, and then just use it here and below